nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -38,6 +38,19 @@
   If you experience any issues, please report them.
   The original Perl script can still be used for now by setting `system.switch.enableNg` to `false`.
 
+- The [Xen Hypervisor](https://xenproject.org) is once again available as a virtualisation option under [`virtualisation.xen`](#opt-virtualisation.xen.enable).
+  - This release includes Xen [4.16.6](https://wiki.xenproject.org/wiki/Xen_Project_4.16_Release_Notes), [4.17.4](https://wiki.xenproject.org/wiki/Xen_Project_4.17_Release_Notes), [4.18.2](https://wiki.xenproject.org/wiki/Xen_Project_4.18_Release_Notes) and [4.19.0](https://wiki.xenproject.org/wiki/Xen_Project_4.19_Release_Notes), as well as support for booting the hypervisor on EFI systems.
+    ::: {.warning}
+      Xen 4.16 is due to be **deprecated** on [2024-12-02](https://xenbits.xenproject.org/docs/unstable/support-matrix.html).
+    :::
```

Comment:
I still think we really ought not to ship this essentially born‐deprecated package to the stable channel. If the release slips it could even come out *after* it leaves support.

Reply:
#340609